client/static/js/app.js

```diff
@@ -1,14 +1,14 @@
 
 $(document).ready( function() {
-    var board = Board.init();
+    var Board = board.init();
```

Comment:
it was a quick fix since everything i programmed referenced board and not Board, i would have had to change everything.